WordPress/gutenberg
# Block List: Display inserter button after block on hover, focus

Closes #2752
Related: #833

This pull request seeks to display a block inserter after a hovered or focused block, enabling a user to easily insert between blocks. This is a first step toward eventually dropping the header global inserter (#2755).

![image](https://user-images.githubusercontent.com/1779930/31246115-5dddf6f6-a9da-11e7-8da1-deabd8c1286c.png)

__Design notes:__

This is a best-attempt design. Iterations or suggestions are welcome. In particular, the specifics of the clickable/tappable area and hover bounds may need some improvement.

__Testing instructions:__

Verify that you can insert after a block which is focused or hovered, and that regressions do not otherwise occur with the global inserter or end-of-content inserters.

1. Navigate to Gutenberg > New Post
2. Insert a paragraph
3. Note that hovering this paragraph exposes no inserter, since it's the last block (expected to use end-of-content inserter)
4. Insert a second block
5. Hover and/or focus the first paragraph inserted in step 2
6. Note that an inserter is shown, enabling you to insert a new block after the paragraph
